### PR TITLE
Create RemoteDesktopServices-RdpCoreTS-parser.txt

### PR DESCRIPTION
**Pull request by Molx32**
Fixes #

## Proposed Changes

  - Parser for Microsoft-Windows-RemoteDesktopServices-RdpCoreTS/Operational Windows events channel. This might be usefull to detect RDP brute force attacks.

  -
  -

### CODE REVIEW

**Comment by shainw** [comment]
Will review over the next week.  Need to turn on these events in my test environment to check them out and be able to comment. :)

**Comment by Molx32** [comment]
@shainw thank you! Let me know if you need anything to test it :)

**Comment by ashwin-patil** [comment]
Hey @Molx32 , thank you for the contribution. 

I tested the parser and it works fine for the events included in the parser but **needs more coverage for additional events** as EventProvider- _Microsoft-Windows-RemoteDesktopServices-RdpCoreTS/Operational_ has more than 50 events. The current parser only parses selected events but the original event provider had more type of events. If we use this parser function, we will not see those events triggered from original provider.

Couple of options i think , we can either increase the coverage of this parser for all the supported events, this will be time consuming.  so let`s make sure all the relevant events needed from detection perspective are parsed and create container for all unparsed events and include in the union at the end. 

e.g. 
```
let RemoteDesktopServicesUnparsed=()
{
    let RemoteDesktopServicesUnparsed= RemoteDesktopServices
    | where EventID !in (65,72,102,103,131,140,141,142,143,145,148,226,227)
}
```

If you want to take a look at all the events associated with this provider, you can check below 2 resources:
- OSSEM Data Dictionary - [Microsoft-Windows-RemoteDesktopServices-RdpCoreTS ETW events](https://github.com/hunters-forge/OSSEM/tree/master/data_dictionaries/windows/etw-providers/Microsoft-Windows-RemoteDesktopServices-RdpCoreTS)
- [Windows Event Provider Explorer (WEPExplorer)](https://lallouslab.net/2016/01/25/windows-events-providers-explorer/)

Reference: 
https://docs.microsoft.com/en-us/message-analyzer/system-etw-provider-event-keyword-level-settings#finding-system-etw-provider-metadata

**Comment by shainw** [comment]
@Molx32 - Please have a look at @ashwin-patil comments.  Thanks, Shain.

**Comment by Molx32** [comment]
Sorry for my lack of answer, I didn't have much time to spend on the topic. I will update the parser in the next few days. By the way, thank you @ashwin-patil for your complete and helpful answer.

**Comment by Molx32** [comment]
I am currently investigating on events that may be useful from a security detection / analysis point of view. I put this [sheet](https://metsysgroup-my.sharepoint.com/:x:/g/personal/clement_bonnet_metsys_fr/EWuFpCP_YWpAv8Nx4ctBR3IBk3iIHGC7FA1abBeEjORH0w?e=tCLWw0) here for people that would be interested in events and their description.

File: Parsers/RemoteDesktopServices-RdpCoreTS-parser.txt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+// This might be usefull to detect success bruteforce attacks on RDP.
+// Note that this parser was initially created to monitor EventID 140 (RDP failed auth.). Thus, at the moment, not all EventID are supported.
+// Don't hesitate to update it or to ask for an update.
+//
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias (e.g. RDPCoreTS).
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. RDPCoreTS | where EventID == 140).
+//
+// References : 
+// Using functions in Azure monitor log queries : https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/functions
```
Comment (ashwin-patil) [inline Parsers/RemoteDesktopServices-RdpCoreTS-parser.txt:15]: We recently updated all our doc links to remove locale en-us. 
Can you just replace this url without locale :  https://docs.microsoft.com/azure/azure-monitor/log-query/functions

**Pull request merged**